src/Modules/BuildList.lua

```diff
@@ -141,7 +156,14 @@ end
 
 function listMode:BuildList()
 	wipeTable(self.list)
-	local handle = NewFileSearch(main.buildPath..self.subPath.."*.xml")
+	-- local handle = NewFileSearch(main.buildPath..self.subPath.."*.xml")
+	local filterList = main.filterBuildList or ""
+	local handle = nil
+	if filterList ~= "" and filterList ~= nil then
```

Comment:
`filterList` will never be `nil` here, so the extra check is unneeded.

Reply:
Ok! Removed it